packages/hint-no-html-only-headers/src/hint.ts

```diff
@@ -37,12 +37,23 @@ export default class NoHtmlOnlyHeadersHint implements IHint {
             'content-security-policy',
             'feature-policy',
             'x-content-security-policy',
-            'x-frame-options',
             'x-ua-compatible',
             'x-webkit-csp',
             'x-xss-protection'
         ];
 
+        const exceptionHeaders: string[] = [
```

Comment:
Nit: we don't need to specify a type here (feel free to remove the existing redundant type for `unneededHeaders` too).

```suggestion
        const exceptionHeaders = [
```